apps/delivery-options/src/composables/useFeatures.ts

```diff
@@ -1,19 +1,22 @@
 import {computed} from 'vue';
 import {useMemoize} from '@vueuse/core';
-import {ConfigSetting, PACKAGE_TYPE_DEFAULT} from '@myparcel-do/shared';
+import {ConfigSetting} from '@myparcel-do/shared';
+import {PackageTypeName} from '@myparcel/constants';
 import {useConfigStore} from '../stores';
 import {SHOWN_SHIPMENT_OPTIONS} from '../data';
 
+const PACKAGE_TYPE_DEFAULT = [PackageTypeName.Package, PackageTypeName.PackageSmall];
```

Comment:
This looks good, but I'd suggest renaming this to avoid confusion with `PACKAGE_TYPE_DEFAULT` from the shared lib (which would require a bigger refactor if *that* changed)

Perhaps like this would be more clear;
```suggestion
const SUPPORTED_PACKAGE_TYPES = [PackageTypeName.Package, PackageTypeName.PackageSmall];
```

Otherwise this looks good so I'll make this a non-blocking comment :)

Reply:
Thank you for the great suggestion I made the changes accordingly.